ceph-installer-merged/config/definitions/ceph-installer-merged.yml

```diff
@@ -0,0 +1,47 @@
+- job:
+    name: ceph-installer-merged
```

Comment:
I was thinking of making this the "when it merges to master" job, but happy to update the name if you think it shouldn't have anything else piled onto it